packages/ci/package.json

```diff
@@ -33,8 +36,10 @@
     "@types/mocha": "^5.2.6",
     "@types/nock": "^9.3.1",
     "@types/node": "^10.12.24",
+    "@types/phoenix": "^1.4.0",
```

Comment:
Were you not able to get rid of `socket.io-client`?

Reply:
no, this PR keeps the socket.io stuff around unless the 2 environment variables are set to try the new behavior. I removed socket.io from the ci-v5 plugin though because it wasn't actually used.